### PR TITLE
Add entry quantity adjustment buttons

### PR DESCRIPTION
**Pull request by achiu8**

### CODE REVIEW

File: src/components/MasterItems.js [hunk 1]
```diff
@@ -73,21 +78,37 @@ class MasterItems extends Component {
...
+            className: `input form-control add-comment-${entry.name}`,
+            onChange: this.addComment.bind(null, i),
+            placeholder: 'Comment',
+            onKeyPress: this.handleKeyPress,
+            value: entry.comment
+          }
+        )
+      ),
+      dom.td(
+        { className: 'entry-quantity col-xs-3 row' },
```
Comment (kprakobkit) [inline src/components/MasterItems.js]: Can we do bigger? It's showing up all messed up on the ipad.
img_0054

File: src/components/MasterItems.js [hunk 2]
```diff
@@ -56,8 +56,13 @@ class MasterItems extends Component {
     this.setState({ entries: updatedEntries });
   }
 
-  setQuantity(entryIndex, e) {
-    const updatedEntry = R.assoc('quantity', e.target.value, this.state.entries[entryIndex]);
+  setQuantity(entryIndex, change) {
+    const entry = this.state.entries[entryIndex];
+    const newQuantity = entry.quantity + change;
+
+    if (newQuantity < 1) return;
```
Comment (kprakobkit) [inline src/components/MasterItems.js:63]: Should we also test this?

**Comment by achiu8** [comment]
Sizing of quantity / buttons looks kinda janky.. but it shouldn't wrap now.

File: src/components/MasterItems.js (hunk 2 above)
Comment (achiu8) [inline src/components/MasterItems.js:63]: Added.

**Pull request merged**